### PR TITLE
chore: run upgrade test on push to main

### PR DESCRIPTION
**Pull request by tomjohnburton**

### CODE REVIEW

File: .github/workflows/ci-main.yml [hunk 1]
```diff
@@ -58,3 +58,10 @@ jobs:
       version: ci/${{ github.sha }}/
       environment: "development"
     secrets: inherit
+  upgrade-check:
+    needs: [build]
```
Comment (kylezs) [inline .github/workflows/ci-main.yml:62]: needs [build-try-runtime]

Comment (tomjohnburton) [inline .github/workflows/ci-main.yml:62]:
```suggestion
    needs: [build-try-runtime]
```

**Comment by kylezs** [comment]
Also realised this after seeing two jobs fail earlier - we probably can disable it on nightly now.

**Comment by tomjohnburton** [comment]
Removed from nightly tests

**Comment by kylezs** [comment]
Probably best to leave this PR on standby until the upgrade-test is working on main

**Review by kylezs — APPROVED** [review]

**Pull request merged**